modules/virt-creating-offline-vm-snapshot-web.adoc

```diff
@@ -1,12 +1,21 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
 
-[id="virt-creating-offline-vm-snapshot-web_{context}"]
-= Creating an offline virtual machine snapshot in the web console
+[id="virt-creating-vm-snapshot-web_{context}"]
+= Creating a virtual machine snapshot in the web console
 
 You can create a virtual machine (VM) snapshot by using the web console.
 
+[IMPORTANT]
+====
+In order to create snapshots of an online (Running state) VM, you must install the QEMU guest agent.
```

Comment:
same as above.

Reply:
New version (NOTE) in all the renamed files without OFFLINE in the file name.